azure-pipelines.yml

```diff
@@ -130,15 +130,14 @@ jobs:
       skipTests: true
       targetArchitecture: armel
 
-# - template: /eng/jobs/linux-build.yml
-#   parameters:
-#     additionalMSBuildArgs: /p:OutputRid=rhel.6-x64
-#     displayName: Build_Linux_x64_FreeBSD
-#     dockerImage: microsoft/dotnet-buildtools-prereqs:centos-6-376e1a3-20174311014331
-#     osGroup: FreeBSD
-#     portableBuild: true
-#     skipTests: true
-#     targetArchitecture: x64
+- template: /eng/jobs/linux-build.yml
+  parameters:
+    displayName: Build_Linux_x64_FreeBSD
```

Comment:
Per the mail thread, this leg will have to be internal-only for now: there's no publicly available pool for FreeBSD yet, only the internal one. (I expect FreeBSD legs in CI would fail if we reran it right now.)

Reply:
>Per the mail thread, this leg will have to be internal-only for now: there's no publicly available pool for FreeBSD yet, only the internal one. (I expect FreeBSD legs in CI would fail if we reran it right now.)

I pushed another change to enable FreeBSD leg for internal builds only.

Reply:
Ah, yeah, I missed that it changes the basics depending on what's running:

https://github.com/dotnet/core-setup/blob/a8f4ea60d6bb22762777c0097e7dbfeef45a355c/eng/jobs/linux-build.yml#L35-L43

`bash-build.yml` then, I guess?